crates/nu-protocol/src/cli_error.rs

```diff
@@ -11,6 +11,13 @@ pub struct CliError<'src>(
     pub &'src StateWorkingSet<'src>,
 );
 
+pub fn format_error(
+    working_set: &StateWorkingSet,
+    error: &(dyn miette::Diagnostic + Send + Sync + 'static),
+) -> String {
+    return format!("Error: {:?}", CliError(error, working_set));
```

Comment:
minor nit: a little surprised clippy didn't complain that you used a return here

Reply:
Good catch, old habits die hard. I'll clean that up next time I come back to this area.